src/diamonds/nayms/facets/MarketFacet.sol

```diff
@@ -52,8 +51,7 @@ contract MarketFacet is Modifiers, ReentrancyGuard {
         bytes32 _sellToken,
         uint256 _sellAmount,
         bytes32 _buyToken,
-        uint256 _buyAmount,
-        uint256 _feeSchedule
```

Comment:
Given that this is removed, where are we testing the `PLATFORM_ACTION` action fee schedule? I don't think we are, which might be why coverage isn't hitting that branch

Reply:
With current implementation `PLATFORM_ACTION` is not testable, we don't have it implemented. There is no place in our contracts where the system creates an offer yet. I spoke to @tgeorgas about it and we decided to lave it at this for the time being, until the system offers are implemented. It would not be possible to create it by calling the facet with the system admin account passing this fee schedule in as an argument. We don't have an entity to which the system admin belongs which would be making en offer. Now all offers are created on behalf of the msg.sender's related entity.

Reply:
Ok, let's leave it as is